mne/preprocessing/ica.py

```diff
@@ -271,7 +272,8 @@ class ICA(ContainsMixin):
     Methods currently implemented are FastICA (default), Infomax, and Picard.
     Standard Infomax can be quite sensitive to differences in floating point
     arithmetic. Extended Infomax seems to be more stable in this respect,
-    enhancing reproducibility and stability of results.
+    enhancing reproducibility and stability of results; use Extended Infomax
+    via ``method='infomax', fit_params=dict(extended=True)``.
 
     Reducing the tolerance (set in `fit_params`) speeds up estimation at the
```

Comment:
As a user, I would like to get some more information on which paramters I can pass to `fit_params`. If the ICA docstring is not the right place for this, perhaps you can add a pointer to where this is described instead